focal/4_gs1_identification_examples.html

```diff
@@ -0,0 +1,94 @@
+<section>
+      <h3>Focal Use Case: Chain of GS1 Credentials to Identify a Trade Item</h3>
+
+      <p>
+            These examples are based on the document <a href="https://ref.gs1.org/gs1/vc/data-model/" rel="nofollow">"GS1 Verifiable Credentials - Data Model and Validations"</a>. The examples use Verifiable Credentials Data Model 2.0.
+      </p>
+      <p>
+            The key artefact is the last one; it declares the existence of a GTIN, around which other Verifiable Credentials may be issued to declare data about the trade item (brand, size and unit of measure, ingredients, dimensions and weights, etc.). Validation of the artefact requires validating all the credentials that come before it, identified in each case by "extendsCredential".
+      </p>
+      
+      <pre class="example" title="GS1 Prefix 952 Licensed by GS1 Global Office to GS1 Utopia">
+{
+      "@context": [
+            "https://www.w3.org/2018/credentials/v1",
+            "https://ref.gs1.org/gs1/vc/license-context/"
+      ],
+      "id": "did:example:079713e6-9646-4de1-a89d-573dfbd77968",
+      "type": [
+            "VerifiableCredential",
+            "GS1PrefixLicenseCredential"
+      ],
+      "issuer": "did:web:id.gs1.org",
+      "validFrom": "2005-01-01T00:00:00Z",
+      "credentialSubject": {
+            "id": "did:web:www.gs1ut.org",
```

Comment:
I know this is ficticious GS1 utopia, but it might be hard to follow the example of you don't make this inference.  I think a comment that we will use a fictitious gs1ut.org when we introduce the GS1 Utopia analogy above.

This is only 1 letter off from gs1us.org and might cause confusion.

Reply:
Replaced gs1ut.org with gs1utopia.example and replaced example.ut with healthytots.example.